js/packages/teams-ai/temp/teams-ai.api.md

```diff
@@ -0,0 +1,438 @@
+## API Report File for "@microsoft/teams-ai"

```

Comment:
I don't think we want to use a directory called temp. Could we move these and related files to the `.github` folder?

Reply:
Or was this unintentionally checked in?

Reply:
It was generated by `api-extractor` with the default values. I've changed them to generate all API report files under `packages` root folder. Feel free to change the report folder in the base `api-extractor.json` configuration file.